lms/djangoapps/mobile_api/users/views.py

```diff
@@ -396,6 +517,8 @@ def paginator(self):
 
         if self._paginator is None and api_version == API_V3:
             self._paginator = DefaultPagination()
+        if self._paginator is None and api_version == API_V4:
```

Comment:
We need to use something like this here:
`if self._paginator is None and api_version == API_V4:`
Because the method `paginator` is called 4 times when we requesting the API. And only the first time we don't have pagination object. Then we have pagination object:
`<lms.djangoapps.mobile_api.users.views.UserCourseEnrollmentsV4Pagination object at 0x7f57017f5950>`
And we can't change pagination object, we need only return pagination object for the next method calls.